### PR TITLE
CH ZRH EURD

### PR DESCRIPTION
**Pull request by bsaxpi**
PR for additional fields, abstracts, entity types etc. for the CH ZRH EURD building

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/HWS.yaml [hunk 1]
```diff
@@ -229,3 +229,24 @@ HWS_US_MTV_2000_2:
...
+  implements:
+  - HWS
+  - SWTC
+  - CPC
+  uses:
+  - circulation_pump_speed_percentage_command
+  - circulation_pump_speed_percentage_setpoint
+  - failed_alarm
+
+HWS_NON_CANONICAL_TYPE_1:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/HWS.yaml:246]: We dont need to call these out as incomplete here, you would translate them incompletely. This thing would simply be HWS_SWTC and would be canonical; the specific entity in building config would implement HWS_SWTC and have missing fields if they weren't there.

File: ontology/yaml/resources/states/states.yaml [hunk 2]
```diff
@@ -56,7 +56,9 @@ STANDBY: "Waiting in a prepared state."
 REMOTE: "The control is operated remotely."
 LOCAL: "The control is operating locally and independently with the remote system."
 
-
+#New states for CH ZRH eurd
+FADING: "An operation mode of a lighting dimmer"
```
Comment (tasodorff) [inline ontology/yaml/resources/states/states.yaml:60]: Expand definition to distinguish from ramping. Fading goes from high illuminance to low?

File: ontology/yaml/resources/states/states.yaml [hunk 3]
```diff
@@ -56,7 +56,9 @@ STANDBY: "Waiting in a prepared state."
...
+RAMPING: "An operation mode of a lighting dimmer"
```
Comment (tasodorff) [inline ontology/yaml/resources/states/states.yaml:61]: Expand to distinguish from fading; low to high illuminance??

**Review by tasodorff — CHANGES_REQUESTED** [review]
A few minor changes requested.

**Comment by bsaxpi** [comment]
> A few minor changes requested.

Changes done, conflicts resolved

**Review by tasodorff — APPROVED** [review]

**Pull request merged**